articles/techniques/local-variables.md

```diff
@@ -69,7 +69,7 @@ For this reason, it is a good practice to make functions and operations which us
 
 ## Deconstruction ##
 
-In addition to assigning a single variable, the `let` and `mutable` keywords - or in fact any other binding construct - also allow for unpacking the contents of a [tuple type](#tuple-types).
+In addition to assigning a single variable, the `let` and `mutable` keywords - or in fact any other binding construct - also allow for unpacking the contents of a tuple type.
```

Comment:
Why was this link simply removed?

Reply:
The heading that is supposed to link to doesn't exist, apparently removed sometime earlier.

Reply:
I believe the section still exists: https://docs.microsoft.com/en-us/quantum/language/type-model?view=qsharp-preview#tuple-types ?